lib/install/web.rb

```diff
@@ -1,13 +1,19 @@
 def node_version
-  ENV["NODE_VERSION"] || `node --version`[/\d+\.\d+\.\d+/]
+  version = ENV["NODE_VERSION"] || `node --version`[/\d+\.\d+\.\d+/]
+
+  return if version.blank?
+
+  Gem::Version.new(version)
```

Comment:
Would the following work?

```suggestion
  if version = ENV.fetch(ENV["NODE_VERSION"], `node --version`[/\d+\.\d+\.\d+/])
    Gem::Version.new(version)
  else
    ""
  end
```

It solves a couple of issues.

1. Deals with `#[]` returning something falsy but not nil
2. Gets rid of the trailing conditional and makes the method more idiomatic
3. Ensures the method always returns a string